File: individual-projects/2023/amoli-patel-200103102100/Fundamental Practice/if else switch/pen.js
// Check all pens in the box are working or not. If not working then remove it from box and print total working pens.
function main() {
    let working = 0,
        not_working = 2 // assume pen not working
    for (pens = 1; pens <= 5; pens++) {
        if (pens === not_working) {
            console.log('removed')
            continue
        }
        console.log('pen', pens, 'is working')
        working++
    }
    console.log('total working pens are', working)
}
main()

File: individual-projects/2023/ronak-bagdai-200103041003/Fundamental-Practice/If-Else/Pen.rs
fn remove_pen(){
    remove faulty_pens;
}

fn main() {
    let n = 10;
    let working_pens = 0;
    let faulty_pens = 0;

    for i in 1..=n {
        if i is working {
            working_pens +=1;
        }
        else {
            faulty_pens +=1;
        }
    }
    remove_pen();
    print!("Number of Working Pens are {}", working_pens);
}